### PR TITLE
Package renaming and build fix

### PR DESCRIPTION
**Pull request by fmirus**
### Description
major renaming to make naming convention more similar to pytrees:
 - scenario_execution_base --> scenario_execution
 - scenario_execution --> scenario_execution_ros

### Pull request checklist

- [ ] [Issue](https://github.com/IntelLabs/scenario_execution/issues) created that explains the change and why it's needed
- [ ] Tests are part of the PR (for bug fixes / features)
- [x] [Docs](https://intellabs.github.io/scenario_execution/) reviewed and added / updated if needed (for bug fixes / features)
- [x] Format and Lint checks (`make check`) pass locally
- [x] PR applies Apache 2.0 License to all code files

### Pull request type

- [x] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [x] Documentation changes

### CODE REVIEW

File: scenario_execution/test/test_scenario_oneof.py [hunk 1]
```diff
@@ -20,10 +20,10 @@
 import unittest
 from datetime import datetime
 
-from scenario_execution_base.scenario_execution import ScenarioExecution
-from scenario_execution_base.model.osc2_parser import OpenScenario2Parser
-from scenario_execution_base.model.model_to_py_tree import create_py_tree
-from scenario_execution_base.utils.logging import Logger
+from scenario_execution.scenario_execution_base import ScenarioExecution
```
Comment (fred-labs) [inline scenario_execution/test/test_scenario_oneof.py:23]: _base?

File: scenario_execution/test/test_scenario_parallel.py [hunk 2]
```diff
@@ -19,10 +19,10 @@
 """
 import unittest
 
-from scenario_execution_base.scenario_execution import ScenarioExecution
-from scenario_execution_base.model.osc2_parser import OpenScenario2Parser
-from scenario_execution_base.model.model_to_py_tree import create_py_tree
-from scenario_execution_base.utils.logging import Logger
+from scenario_execution.scenario_execution_base import ScenarioExecution
```
Comment (fred-labs) [inline scenario_execution/test/test_scenario_parallel.py:22]: _base?

File: scenario_execution_ros/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Scenario Execution
```
Comment (fred-labs) [inline scenario_execution_ros/README.md:1]: ROS

**Review by fred-labs — CHANGES_REQUESTED** [review]

File: scenario_execution/test/test_scenario_parallel.py (hunk 2 above)
Comment (fmirus) [reply scenario_execution/test/test_scenario_parallel.py:22]: unfortunately, the name of the file cannot be the same as the package (which results in errors), hence the name of the script remained scenario_execution_base (other suggestions welcome 😄)

File: scenario_execution/README.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
-# Scenario Execution Base Package
+# Scenario Execution Package
```
Comment (fred-labs) [inline scenario_execution/README.md:1]: remove "package"? To be consistent with others

**Review by fred-labs — CHANGES_REQUESTED** [review]

**Review by fred-labs — APPROVED** [review]

**Pull request merged**